plugins/woocommerce-admin/client/products/sections/images-section.scss

```diff
@@ -43,6 +43,17 @@
 			}
 		}
 	}
+	&__images:not(.has-images) {
+		.woocommerce-form-file-upload {
+			display: flex;
+			justify-content: center;
+			margin: -4em -2.5em -2.5em;
```

Comment:
I'm a little hesitant on adding negative margins.  This currently goes outside the parent container a bit.

Could we set `.components-card__body` to `0` padding when there are no images?  We'll also need to set the margin on `.woocommerce-sortable` to `0` when there are no images.